src/jest.res

```diff
@@ -357,9 +357,9 @@ module Expect = {
 
   /* toHaveBeenCalled* */
 
-  let toBeCloseTo = (b, p) => FloatCloseTo(mapMod(a => (a, b, None), p))
+  let toBeCloseTo = (. b, p) => FloatCloseTo(mapMod(a => (a, b, None), p))
 
-  let toBeSoCloseTo = (b, ~digits, p) => FloatCloseTo(mapMod(a => (a, b, Some(digits)), p))
+  let toBeSoCloseTo = (. b, ~digits, p) => FloatCloseTo(mapMod(a => (a, b, Some(digits)), p))
```

Comment:
I don't get why these need to be uncurried. These are normal functions, not JS interop. Are you sure they need to be?

Reply:
I'm relatively new to Rescript and not quite sure why they fail when curried.  I stepped through the curried version of these and found the failure in Recript's curry.js (see [https://github.com/rescript-lang/rescript-compiler/issues/5326](https://github.com/rescript-lang/rescript-compiler/issues/5326)).  Perhaps you with your greater experience with BuckleScript might suggest an alternative.

Reply:
I understand this now.  Rescript 9.x includes an @return(nullable) decorator to handle situations where a curried function call returns undefined.  Forced uncurrying of these calls address the issue but it should be possible to modify your code to handle the undefined return value specifically.  More work required. ...

Reply:
**Final:** It is not pretty, but I wrapped toBeCloseTo and toBeSoCloseTo so that user' of the API don't have to uncurry call sites to these.

Reply:
See suggestions below